app/assets/stylesheets/components/modals.scss

```diff
@@ -2,25 +2,21 @@
 
 .crayons-modal {
   --modal-max-width: 768px;
-  --modal-body-padding: var(--su-3);
-  --modal-header-padding: var(--su-2) var(--su-2) var(--su-2) var(--su-3);
-  --modal-header-font-size: var(--fs-l);
-  --modal-body-font-size: var(--fs-base);
+  --modal-padding: var(--su-3);
```

Comment:
Lots of variables here but it's either THAT or having lots of utility classes in the component itself. If we had only Preact components in use this wouldn't be an issue, but since we also have HTML version of these components I decided to "hide" everything in CSS where devs won't be faced with some extra classes.

Reply:
Just curious where we have modals in HTML still. I thought they all launch the Preact component now?

Reply:
Somewhere on user profile, some liquid tag (that was actually a surprise for me) but we also use some in vanilla JS and even in preact components (onboarding - which I have no idea why it doesn't use `<Modal>` component and it needs to be fixed... but its just sooo customized.. sigh..) and even looks like some stuff in [admin](https://github.com/forem/forem/blob/main/app/javascript/admin/adminModal.js#L34)...

Reply:
I'll create an issue to convert all these to the Preact modal component. It's not just for cleanup, it also gives us a tonne of accessibility baked in.

Reply:
I created #16328 for this.